### PR TITLE
Initial commit of an exim role (tested)

### PR DESCRIPTION
**Pull request by Scriptkiddi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Role Name
```
Comment (SlothOfAnarchy) [inline README.md]: .

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+This role configures and installs exim.
+
+## Requirements
+
+This role only needs an apt based system e.g. Ubuntu
+
+## Role Variables
+Every section relates to similar named section in a normal [exim configuration](https://www.exim.org/exim-html-current/doc/html/spec_html/index.html).
```
Comment (SlothOfAnarchy) [inline README.md:10]: to a similarily named section?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+This is the reason why there is no desription for these section.
```
Comment (SlothOfAnarchy) [inline README.md]: des**c**ription

File: README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+| `exim_acls_section`          | ` `                 |
+| `exim_router_section`        | ` `                 |
+| `exim_transport_section`     | ` `                 |
+| `exim_authenticator_section` | ` `                 |
+| `exim_rewrite_section`       | ` `                 |
+| `exim_retry_section`         | ` `                 |
+
+## Example Playbook
+
+Please see the excellent exim documentation on how to file the sections.
```
Comment (SlothOfAnarchy) [inline README.md]: fill

File: meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Fritz Otlinghaus
+  description: This role installs and configure exims
```
Comment (SlothOfAnarchy) [inline meta/main.yml]: configures

File: meta/main.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: CC-BY-SA
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Ubuntu
```
Comment (SlothOfAnarchy) [inline meta/main.yml:13]: what about debian?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+## Role Variables
+Every section relates to similar named section in a normal [exim configuration](https://www.exim.org/exim-html-current/doc/html/spec_html/index.html).
+This is the reason why there is no desription for these section.
+
+| Variable                     | Default / Mandatory |
+|------------------------------|---------------------|
+| `exim_local_section`         | :heavy_check_mark:  |
+| `exim_main_section`          | :heavy_check_mark:  |
+| `exim_acls_section`          | ` `                 |
```
Comment (SlothOfAnarchy) [inline README.md]: the default is not a space. Just leave the fields empty in this case.

File: templates/exim.conf.j2 [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+{{ exim_transport_section | default("") | indent( width=8, indentfirst=True) }}
+
+######################################################################
+#                     AUTHENTICATORS CONFIGURATION                   #
+######################################################################
+
+begin authenticators
+
+{{ exim_authenticator_section | default("") | indent( width=8, indentfirst=True) }}
```
Comment (SlothOfAnarchy) [inline templates/exim.conf.j2:46]: You could mention in your readme that you expect a (multi-line) string for each section.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Scriptkiddi) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (Scriptkiddi) [reply README.md:10]: similar works here

File: README.md (hunk 3 above)
Comment (Scriptkiddi) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (Scriptkiddi) [reply README.md]: fixed

File: meta/main.yml (hunk 6 above)
Comment (Scriptkiddi) [reply meta/main.yml:13]: done

File: templates/exim.conf.j2 (hunk 8 above)
Comment (Scriptkiddi) [reply templates/exim.conf.j2:46]: fixed

**Comment by Scriptkiddi** [comment]
@SlothOfAnarchy pls rereview

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**